core/sail/api/src/main/java/org/eclipse/rdf4j/sail/helpers/AbstractSailConnection.java

```diff
@@ -973,6 +971,13 @@ protected void prepareInternal() throws SailException {
 	protected abstract void addStatementInternal(Resource subj, IRI pred, Value obj, Resource... contexts)
 			throws SailException;
 
+	protected void bulkAddStatementsInternal(final Collection<? extends Statement> statements)
```

Comment:
Could you annotate this with @Experimental so that we can adjust the signature down the line if we need to?

Reply:
Sure thing!